src/game.zig

```diff
@@ -332,6 +332,59 @@ pub const collision = struct {
 		return @floatCast(friction/totalArea);
 	}
 
+	pub fn calculateBounce(comptime side: main.utils.Side, pos: Vec3d, hitBox: Box) f32 {
```

Comment:
I assume this is a copy of the friction calculation?
Please merge these functions, as is done for the volume calculations.